google-account-plugin/src/com/google/cloud/tools/intellij/login/GoogleLoginPrefs.java

```diff
@@ -82,17 +86,23 @@ public static void saveOAuthData(OAuthData credentials) {
    * an {@code OAuthData} object all of whose getters return {@code null} .
    */
   public static OAuthData loadOAuthData() {
-    Preferences prefs = getPrefs();
+    String refreshToken = null;
+    String storedEmail = null;
+    SortedSet<String> storedScopes = new TreeSet<String>();
```

Comment:
I guess the old code returns the empty set, so it makes sense not to change  that behavior.